router/src/components/link.rs

```diff
@@ -71,6 +75,10 @@ pub fn A<H>(
 where
     H: ToHref + 'static,
 {
+    #[cfg_attr(
```

Comment:
Does the inner function need to be instrumented separately? I believe you if it does. Can leave at trace.